forgot_password/__init__.py

```diff
@@ -194,3 +193,13 @@ def reset_password_handler(request):
             return skygear.Response(body, content_type='text/html')
 
     return reset_password_response(**template_params)
+
+
+def init():
```

Comment:
The real problem is that we want to define our function, using decorator (the recommended approach), and we want the developer in control of whether registration takes effect. I would like to propose that we use flask’s blueprint mechanism.

Basically:

```
blueprint = skygear.create_blueprint('blueprint-name')  # or skygear.Blueprint('blueprint-name')

@blueprint.handler('handler-name')
def handler_func(request):
    pass

# to enable the blueprint:
if enable:
    skygear.enable_blueprint('blueprint-name')
# or:
# blueprint enabled automatically if some environment variable exists
```

The above handler will only be registered if the blueprint is enabled.